src/gauss.rs

```diff
@@ -58,9 +58,8 @@ fn plx(ell: i64, x: Df64) -> (Df64, Df64)
     // Bonnet's recursion formula
     for k in 1..ell {
         // next term
-        let kk = k as f64;
-        let p2 = ((2.0 * kk + 1.0) * x * p1 - kk * p0) / (kk + 1.0);
-
+        let kk: Df64 = Df64::from(k as f64);
+        let p2 = ((2.0 * kk + Df64::ONE) * x * p1 - kk * p0) / (kk + Df64::ONE);
```

Comment:
This switches from quad-by-double multiplication, which is more accurate and faster, to quad-by-quad multiplication, which is slower and less accurate. Seems to me like not a good idea.

I suspect the test failures are due to numerical noise. Instead of changing the algorithm etc., let's just relax the tolerances slightly.

Reply:
Done